.eslintrc.js

```diff
@@ -26,14 +26,9 @@ module.exports = {
     'processNode': true
   },
   rules: {
-    'ember/no-const-outside-module-scope': 'off',
-    'ember/no-direct-property-access': 'off',
-    'ember/avoid-leaking-state-in-ember-objects': 'warn', // TODO: Remove after upgrading eslint >= 4.2.0
     'ember/no-jquery': 'error',
-    'ember/require-access-in-comments': 'off',
-    'newline-before-return': 'error',
     'no-console': 'off',
-    'one-var': 'off',
+    'no-prototype-builtins': 'off'
```

Comment:
Why are we disabling this rule?

Reply:
There was something violating it, so I disabled it.

Reply:
Just one? Can we just do a line comment to disable it then? There are security implications to this rule, and while I'm not certain they apply to Electron or not, with all the various security risks that have to be carefully monitored/managed in Electron, I think it would be better to keep the rule on.

Reply:
Seems to somehow no longer be an issue